### PR TITLE
:lipstick: [#2081] add profile-edit page designs

### PR DESCRIPTION
**Pull request by jiromaykin**
issue https://taiga.maykinmedia.nl/project/open-inwoner/task/2081 
➕ this PR also solves issue: https://taiga.maykinmedia.nl/project/open-inwoner/task/2082 
~Add back-buttons~ + add correct icon in My profiel + add new designs for edit-profile and view BRP/digid profile-data.
http://localhost:8000/mijn-profiel/
http://localhost:8000/mijn-profiel/edit/
http://localhost:8000/mijn-profiel/mydata/
➕ added new class for fullwidth buttons in forms

Designs: https://www.figma.com/file/iKGhWhstaLIlFSaND2q7cE/OIP---Designs-(new)?type=design&node-id=3%3A6398&mode=design&t=VNpvkCkb7m5lhpIq-1

Note: we do not have Tablet designs yet for any pages with a sidebar...

### CODE REVIEW

File: src/open_inwoner/templates/pages/profile/edit.html [hunk 1]
```diff
@@ -1,45 +1,64 @@
...
-                {% if user.contact_type == "begeleider" %}
-                    {% input form.image no_help=True %}
-                    {% image_crop form.cropping %}
-                {% endif %}
-            {% endif %}
-        {% endwith %}
-        {% form_actions primary_text=_("Opslaan") primary_icon="arrow_forward" secondary_href='profile:detail' secondary_text=_('Terug') secondary_icon='arrow_backward' secondary_icon_position="before" enctype="multipart/form-data" %}
-    {% endrender_form %}
+
+    <div class="profile__grid">
```
Comment (stevenbal) [inline src/open_inwoner/templates/pages/profile/edit.html:17]: This doesn't seem to render properly on mobile devices http://127.0.0.1:8000/mijn-profiel/edit/

Screenshot from 2024-02-20 10-42-43

Is that an oversight or is this out of scope for this PR?

**Review by stevenbal — COMMENTED** [review]
Looks good, just one question about mobile view for profile edit page

**Comment by jiromaykin** [comment]
@stevenbal I removed the back-buttons now because the designer has to update the design; apparently those buttons need to become a single breadcrumb that will just show the 'current' page in the breadcrumb view - but for that we would need to further discuss and open a new issue.

File: src/open_inwoner/templates/pages/profile/edit.html (hunk 1 above)
Comment (stevenbal) [reply src/open_inwoner/templates/pages/profile/edit.html:14]: hmm could be that I didn't have `npm run watch` running, looks good now :+1: 

image

**Review by stevenbal — APPROVED** [review]

**Review by stevenbal — APPROVED** [review]

**Pull request merged**